contrib/windows/tests/tests.py

```diff
@@ -1,413 +1,151 @@
-        runWithPrint(["zpool", "destroy", "-f", "test05"])
-        time.sleep(10)
-        postTest()
-
-        preTest()
-        ret = runWithPrint(["zpool", "create", "-f", "test06", "raidz",
-                            tounc(f1), tounc(f2), tounc(f3)])
-        if ret.returncode != 0:
-            print("FAIL")
-        time.sleep(10)
-        print("Drive letters after pool create:", get_driveletters())
-        runWithPrint(["zpool", "destroy", "-f", "test06"])
-        time.sleep(10)
-        postTest()
-
-        preTest()
-        ret = runWithPrint(["zpool", "create", "-f", "test07", "raidz1",
-                            tounc(f1), tounc(f2), tounc(f3)])
-        if ret.returncode != 0:
-            print("FAIL")
-        time.sleep(10)
-        print("Drive letters after pool create:", get_driveletters())
-        runWithPrint(["zpool", "destroy", "-f", "test07"])
-        time.sleep(10)
-        postTest()
-
-        preTest("snapshot no hang:")
-
-        ret = runWithPrint(["zpool", "create", "-f", "testsn01", tounc(f1)])
-        if ret.returncode != 0:
-            print("FAIL")
-        time.sleep(10)
-        print("Drive letters after pool create:", get_driveletters())
-
-        f = PureWindowsPath(get_driveletters()[0][1], "test01.file")
-        allocate_file(f, 1 * Size.KIB)
-
-        ret = runWithPrint(["zfs", "snapshot", "testsn01@friday"])
-        if ret.returncode != 0:
-            print("FAIL")
-
-        f = PureWindowsPath(get_driveletters()[0][1], "test02.file")
-        allocate_file(f, 1 * Size.KIB)
-
-        ret = runWithPrint(["zpool", "export", "-a"])
-        if ret.returncode != 0:
-            print("FAIL")
-        time.sleep(10)
-
-        runWithPrint(["zpool", "destroy", "-f", "testsn01"])
-
-        time.sleep(10)
-        postTest()
-
-        # preTest("snapshot hang")
-        #
-        # ret = runWithPrint(["zpool", "create", "-f", "testsn02", tounc(f1)])
-        # if ret.returncode != 0:
-        #     print("FAIL")
-        # time.sleep(10)
-        # print("Drive letters after pool create:", get_driveletters())
-        #
-        # f = PureWindowsPath(get_driveletters()[0][1], "test01.file")
-        # allocate_file(f, 1 * Size.KIB)
-        #
-        # ret = runWithPrint(["zfs", "snapshot", "testsn02@friday"])
-        # if ret.returncode != 0:
-        #     print("FAIL")
-        #
-        #
-        # f = PureWindowsPath(get_driveletters()[0][1], "test02.file")
-        # allocate_file(f, 1 * Size.KIB)
-        #
-        # ret = runWithPrint(["zfs", "mount", "testsn02@friday"])
-        # if ret.returncode != 0:
-        #     print("FAIL")
-        #
-        #
-        # ret = runWithPrint(["zpool", "export", "-a"])
-        # if ret.returncode != 0:
-        #     print("FAIL")
-        # time.sleep(10)
-        #
-        # runWithPrint(["zpool", "destroy", "-f", "testsn02"])
-        # time.sleep(10)
-        # postTest()
```

Comment:
Can you recreate this and comment it out?

Reply:
It crashes

Reply:
![Uploading image.png…]()

Reply:
![image](https://github.com/openzfsonwindows/openzfs/assets/922233/6b9ee891-2d1d-4295-87d5-d165986de210)

Reply:
Means you've done it correctly. :)

Reply:
See #289

Reply:
I love you

Reply:
Thanks! I re-added the hanging test but left it commented out.